spec/controllers/polls_controller_spec.rb

```diff
@@ -23,29 +21,27 @@
     end
   end
 
-  describe 'GET #ready_index' do
-    before(:each) do
-      FactoryGirl.create(:valid_poll)
-      FactoryGirl.create(:valid_poll)
-      FactoryGirl.create(:valid_poll, :with_ready_status)
-    end
+  describe 'GET #ready' do
+    let!(:poll1) { FactoryGirl.create(:valid_poll) }
```

Comment:
You can avoid repeating your `let!` blocks if you'll write them outside of `describe 'GET'` block, right in the top-level `describe PollsController` one.

But please keep in mind that this way your `let`s will affect all your tests in this file.
You can find sweet middle ground by using
```ruby
context 'with 3 polls present' do
  let!(:poll1)
  let!(:poll2)
  let!(:poll3)

  describe 'GET'
  describe 'POST'
end
```